HW2/P4/driver.py

```diff
@@ -21,7 +21,16 @@ def py_median_3x3(image, iterations=10, num_threads=1):
     tmpB = np.empty_like(tmpA)
 
     for i in range(iterations):
-        filtering.median_3x3(tmpA, tmpB, 0, 1)
+        threadList = []
```

Comment:
a better speedup could be achieved by using signals and avoiding to start 4 threads each iteration as this is typically very costly